test/TokenTestUtils.js

```diff
@@ -26,34 +28,36 @@ var setAccountDefault = AccountUtils.setAccountDefault;
 var recursiveSetAccountDefault = AccountUtils.recursiveSetAccountDefault;
 var checkState = AccountUtils.checkState;
 var getAccountState = AccountUtils.getAccountState;
+var upperCaseAddress = AccountUtils.upperCaseAddress;
 
 // TODO: test really big numbers  Does this still have to be done??
 
 var adminSlot = "0x10d6a54a4754c8869d6886b5f5d7fbfa5b4522237ea5c60d11bc4e7a1ff9390b";
 var implSlot = "0x7050c9e0f4ca769c69bd3a8ef740bc37934f8e2c036e5a723fd8ee048ed3f8c3";
-const should = require('chai')
-    .use(require('chai-as-promised'))
-    .use(require('chai-bignumber')(BigNumber))
-    .should();
-
-var BigNumber = require('bignumber.js');
 
 // set to true to enable verbose logging in the tests
 var debugLogging = false;
 
+// Returns a new BN object
+function newBigNumber(value) {
+    var hex = new BigNumber(value).toString(16);
+    return new BN(hex, 16);
+}
+
+
 function calculateFeeAmount(amount) {
     return Math.floor((fee / feeBase) * amount);
 }
 
 function checkMinterConfiguredEvent(configureMinterEvent, minter, minterAllowedAmount) {
     assert.equal(configureMinterEvent.logs[0].event, 'MinterConfigured')
-    assert.equal(configureMinterEvent.logs[0].args.minter, minter)
+    assert.equal(upperCaseAddress(configureMinterEvent.logs[0].args.minter), upperCaseAddress(minter));
```

Comment:
Are we able to uppercase only the first argument to assert.equal in these cases since the addresses have been changed to uppercase?